request-manager/request/manager.go

```diff
@@ -420,6 +421,17 @@ func (m *manager) Specs() []proto.RequestSpec {
 				Name:     arg.Name,
 				Desc:     arg.Desc,
 				Required: false,
+				Static:   false,
+				Default:  arg.Default,
+			}
+			s.Args = append(s.Args, a)
+		}
+		for _, arg := range req[name].Args.Static {
```

Comment:
I think it would better to actually for the request manager to not expose static args at all. Since clients can't update them anyway, they don't even need to know about them